### PR TITLE
Po/bd 2659 update dependencies

### PR DESCRIPTION
**Pull request by poborin**

### CODE REVIEW

File: ios/Cartfile.resolved [hunk 1]
```diff
@@ -1 +1 @@
-github "Bluedot-Innovation/PointSDK-iOS" "15.4.0"
+github "Bluedot-Innovation/PointSDK-iOS" "15.4.3"
```
Comment (duncan-lau) [inline ios/Cartfile.resolved:1]: @poborin we just released 15.5.0 :)

Comment (duncan-lau) [reply ios/Cartfile.resolved]: Updated tp 15.5.0

**Review by jsprincep — CHANGES_REQUESTED** [review]
Please aim at dev/philz-fixes. Nothing is to be merged with master until QA is complete.

**Comment by duncan-lau** [comment]
> Please aim at dev/philz-fixes. Nothing is to be merged with master until QA is complete.

Done!

**Review by jsprincep — APPROVED** [review]

**Comment by poborin** [comment]
test comments integration

**Pull request merged**